server/jobHandler/src/jobProcessingEmail.ts

```diff
@@ -0,0 +1,80 @@
+import { getItem, GetItemInput } from "./service/dynamodbClient";
+import { AttributeMap } from "aws-sdk/clients/dynamodb";
+import { getEmail, getJobStatus } from "./service/dynamodbService"
+import * as dotenv from "dotenv";
+dotenv.config(); // configure environment vars
+import * as nodemailer from "nodemailer";
+
+
+/**
+ * Gets a job from the database.
+ * @param request 
+ */
+export const awaitJob = async (request: GetItemInput): Promise<AttributeMap | null> => {
+	const res = await getItem(request);
+	if (res.Item != null) {
+		return res.Item;
+	}
+	return null;
+};
+
+/**
+ * Sends an email to the user telling them that their job is processing.
+ * @param key job key to be accessed from db
+ */
+export const sendProcessingEmail = async (key: string): Promise<any> => {
+    console.log("attempting to send job processing email...");
+    const smtpEndpoint = process.env.SMTP_EMAIL; // aws email endpoint email
+    const port = 465;
+
+	// make request for DB info using job key, store in request
+	const request: GetItemInput = {
+		TableName: process.env.JOB_TABLE as string,
+		Key: {
+			id: { S: key },
+		},
+    };
+
+    // make request, store result in res
+    const res = await awaitJob(request);
+    // get user email 
+    const recipientEmail = await getEmail(key); 
+    // get job status
+    const currJobStatus = await getJobStatus(key);
+    
+    // check that job is processing or generating report
+    if (currJobStatus === "PROCESSING" || currJobStatus === "GENERATING") {
```

Comment:
This check isn't necessary, and it's incorrect. The jobHandler function calls itself, so with this check multiple emails could be sent.